HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -147,6 +150,10 @@ def customiseFor2018Input(process):
 # CMSSW version specific customizations
 def customizeHLTforCMSSW(process, menuType="GRun"):
     
+    # if the gpu modifier is enabled, make the Pixel, ECAL and HCAL reconstruction offloadable to a GPU
+    from HLTrigger.Configuration.customizeHLTforPatatrack import customizeHLTforPatatrack
+    gpu.makeProcessModifier(customizeHLTforPatatrack).apply(process)
```

Comment:
So on non-gpu cases, gpu as python variable exists but makeProcessModifier does nothing?

Reply:
`gpu` exists (it's imported above, at line 4), but it will be disabled if it's not used by the process (like an Era that is not used): in this case the customisation associated to `makeProcessModifier()` is not called.

Reply:
Hi @fwyzard , just wanted to ask a question, for my own education.

I was wondering if one can incur in the case where `customizeHLTforPatatrack` is run twice, now from `customizeHLTforPatatrack` plus once more if the user specifies something like `hltGetConfiguration --customise [..].customizeHLTforPatatrack`. I'm guessing this can't happen because one would not have a way to enable the `gpu` modifier from `hltGetConfiguration`. Or am I missing something? And do you already know if the Patatrack customisations are such that, even if run multiple times, they'd still return the correct configuration?

Reply:
hi Marino,
I have not tired, but based on its implementation I think that trying to apply the `customizeHLTforPatatrack` twice will not work as intended, and/or produce a broken configuration.

Specifying the `gpu` modifier and the `customizeHLTforPatatrack` at the same time would indeed be a mistake - like trying to apply the `customizeHLTforPatatrack` twice.

However, I don't think `hltGetConfiguration` supports process modifiers, so that shouldn't happen.

Reply:
Okay. Thanks for the info!